docker/ssp/config-override.php

```diff
@@ -6,5 +6,6 @@
         'database.username' => 'user',
         'database.password' => 'password',
         'language.i18n.backend' => 'gettext/gettext',
-        'logging.level' => 7
+        'logging.level' => 7,
+//        'usenewui' => true,
```

Comment:
We should probably make a choice here, either true, false or removed completely but not commented out.

Reply:
Ok, I've set it to false by default.